mods/ra2/sequences/misc.yaml

```diff
@@ -277,6 +277,14 @@ moveflsh:
 		Length: *
 		Tick: 30
 
+wake:
+	idle: wake2
```

Comment:
This is wrong. Correct file is the wake1.shp under conquer.mix. There are wake1.shps under other mixes too tho so just writing wake1 doesn't work. This needs a setup like TS cameos use.

Reply:
Corrected that.